src/app/startups/applications/addApplications/page.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
```

Comment:
Please move this logic to 
pages-components folder 
and here just import the component